src/main/java/core/basesyntax/service/BalanceSupplyReturnHandler.java

```diff
@@ -0,0 +1,31 @@
+package core.basesyntax.service;
+
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.model.FruitTransaction.Operation;
+import core.basesyntax.strategy.BalanceSupplyStrategy;
+import core.basesyntax.strategy.OperationStrategy;
+import core.basesyntax.strategy.ReturnStrategy;
+import java.util.HashMap;
+import java.util.Map;
+
+public class BalanceSupplyReturnHandler implements OperationHandler {
```

Comment:
Say we have more Operations, you want to rename this class each time we add more Operations?

Reply:
Don't overcomplicate class. Create handlers for each operation